### DISCUSSION TITLE
There was a problem loading the local develop command. Gatsby may not be installed in your site's "node_modules" directory. Perhaps you need to   run "npm install"? You might need to delete your "package-lock.json" as well.

### DISCUSSION THREAD

**Discussion by danielmurphy1** [gatsbyjs/gatsby#34385] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

Unable to run development. 

After running `gatsby develop` I get the following error:
 "ERROR 

There was a problem loading the local develop command. Gatsby may not be installed in your site's "node_modules" directory. Perhaps you need to  
run "npm install"? You might need to delete your "package-lock.json" as well."

I tried deleting `node_modules` and `package-lock.json` and running `npm install` as well as `npm install --force`  after doing so I get 
![image](https://user-images.githubusercontent.com/62181108/147288795-6c6da460-a09e-46d4-b3d7-0b0ed022dc22.png)
![image](https://user-images.githubusercontent.com/62181108/147288825-fc022b08-c941-4573-aff2-301cda71b562.png)

If I only ran `npm install` and not `npm install --force` the `node_modules` folder that built during the install disappears. I haven't ever seen anything like this.

I have never had a problem before running this command in this project to update the site. 

I have confirmed that I am able to create a new site by running `gatsby new "name-of-test-site"` which starts on port 8000 after running `gatsby develop` in the site directory

### Reproduction Link

https://github.com/danielmurphy1/portfolio-site.git

### Steps to Reproduce

1. Clone my repo https://github.com/danielmurphy1/portfolio-site.git
2. run `npm install` 
3.run `gatsby develop` or `npm run start`

### Expected Result

Site builds in development and is able to be accessed by accessing `http://localhost:8000/` in browser

### Actual Result

Receive "ERROR 

There was a problem loading the local develop command. Gatsby may not be installed in your site's   
"node_modules" directory. Perhaps you need to run "npm install"? You might need to delete your      
"package-lock.json" as well." 

### Environment

```shell
System:
    OS: Windows 10 10.0.19042
    CPU: (4) x64 Intel(R) Core(TM) i5-3570K CPU @ 3.40GHz
  Binaries:
    Node: 16.13.1 - C:\Program Files\nodejs\node.EXE
    npm: 8.1.2 - C:\Program Files\nodejs\npm.CMD
  Languages:
    Python: 3.10.1 - /c/Python310/python
  Browsers:
    Chrome: 96.0.4664.110
    Edge: Spartan (44.19041.1266.0), Chromium (96.0.1054.62)
  npmGlobalPackages:
    gatsby-cli: 4.4.0
    gatsby: 4.4.0
```

**Comment by LekoArts** [comment]
Hi!

It can be multiple things, my best guess would be that you're trying to use a `sass` version that is incompatible with your Node version/setup etc. https://www.npmjs.com/package/node-sass
Hard to tell since the versions are so outdated and `node-sass` itself  is deprecated.
